contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -133,10 +133,9 @@ var EditMetadataView = BaseViews.BaseEditableListView.extend({
   load_editor:function(selected_items){
     var is_individual = selected_items.length === 1;
     var is_exercise = is_individual && selected_items[0].model.get("kind") == "exercise";
-    var has_files = is_individual && (selected_items[0].model.get("assessment_items").length ||
-                    selected_items[0].model.get("files").find(function(f){
-                      return window.formatpresets.get({id:f.preset.name || f.preset.id || f.preset}).get("display");
-                    }));
+    var has_files = is_individual && selected_items[0].model.get("files").some(function(f){
+                      return window.formatpresets.get({id: f.preset.name || f.preset.id || f.preset}).get("display");
```

Comment:
Pretty hard to understand what's going on here too. Why can the `id` of this model be assigned to either a `name`, `id` or `preset` object?